runtime/doc/remote_plugin.txt

```diff
@@ -15,34 +15,34 @@ Nvim support for remote plugins 		  *remote-plugin*
 1. Introduction					    *remote-plugin-intro*
 
 Extensibility is a primary goal of Nvim. Any programming language may be used
-to extend nvim without changes to nvim itself. This is achieved with remote
+to extend Nvim without changes to Nvim itself. This is achieved with remote
 plugins, coprocesses that have a direct communication channel (via
 |msgpack-rpc|) with the Nvim process.
 
-Even though these plugins are running in separate processes they can call, be
-called, and receive events just as if the code was being executed in the main
+Even though these plugins run in separate processes they can call, be called,
+and receive events just as if the plugin's code were executed in the main
 process.
 
+
 ==============================================================================
 2. Plugin hosts					    *remote-plugin-hosts*
 
 While plugins can be implemented as arbitrary programs that communicate
 directly with the high-level Nvim API and are called via |rpcrequest()| and
-|rpcnotify()|, that is not the best approach available. Instead, developers
-should first check if a plugin host implementation is available for their
-chosen programming language.
+|rpcnotify()|, that is not the best approach. Instead, developers should first
+check whether a plugin host is available for their chosen programming language.
```

Comment:
It didn't seem to be adding anything. The rest of the file refers to "plugin hosts," and it sounded fine that way, so I got rid of "implementation" here.